cityofaustin/atd-moped
# Removes deprecated `role_id` column from `moped_proj_personnel`

## Associated issues
Resolves https://github.com/cityofaustin/atd-data-tech/issues/10586

With regard to the activity log—i initially thought the activity log would be impacted by this change, however we are not using "role_id" in the activity log, as Mike documented [here](https://github.com/cityofaustin/atd-data-tech/issues/9423). 

## Testing
**URL to test:** Local

**Steps to test:**
1. Verify that the Hasura cluster starts without error
2. Navigate to any of these project IDs: 176, 188, 182, 174
3. Verify that there are two team members, each with multiple roles.
4. Remove a role from one of these team members
5. Add a new team member.
6. Remove the new team member.
7. Navigate to the main projects list and verify that team members are listed in the "Team" column
8. Navigate to activity log for project 174. Verify that the previous activity renders correctly, notably that "John Clary added Renee O. to the project team".  

---
#### Ship list
